scripts/asm-installer/install_asm

```diff
@@ -1350,6 +1368,17 @@ Authorization: Bearer ${TOKEN}
 EOF
 }
 
+init_meshca_with_hub_idns() {
+  info "Initializing Mesh CA with Hub..."
+  local TOKEN; TOKEN="$(retry 2 gcloud --project="${PROJECT_ID}" auth print-access-token)"
+  run curl --request POST \
+    --data '{"workloadIdentityPools":["'${ENVIRON_PROJECT_ID}'.hub.id.goog"]}' -o /dev/null \
+    "https://meshconfig.googleapis.com/v1alpha1/projects/${PROJECT_ID}:initialize" \
+    --header @- <<EOF
+Authorization: Bearer ${TOKEN}
+EOF
+}
+
```

Comment:
I think we can just have a `get_workload_pool` function which will return a different string based on whether we're using hub or not, then the initialize call logic should be the same.